.github/workflows/ruby.yml

```
@@ -22,6 +22,14 @@ jobs:
         ruby-version: ['2.6', '2.7', '3.0']
         mongodb-version: ['4.4']
         sqlite-version: ['3.0']
+        gemfile:
+          - rails5
+          - rails6.1
+          - mongoid6
+          - mongoid7
+
+    env:
+      BUNDLE_GEMFILE: ${{ github.workspace }}/gemfiles/${{ matrix.gemfile }}.gemfile
```

This can probably be simplified down to:

```suggestion
      BUNDLE_GEMFILE: gemfiles/${{ matrix.gemfile }}.gemfile
```

---

Yes, it is ! I tend to keep this so I don't forget that var name. its only needed when the tests or the app is not at the root but the gemfiles are... it does not hurt to keep, but I have removed.